data/json/stilus.json

```diff
@@ -0,0 +1,249 @@
+                    }
+                ]
+            },
+            {
+                "text": " J. Luginbühl, Salve Domina. Hinweise auf lesende und schreibende Frauen im Römischen Reich, HASBonline 22, 2017, 49–73 (online verfügbar ",
+                "url": [
+                    {
+                        "__type": "URL",
+                        "type": "URL",
+                        "url": "http://dx.doi.org/10.22013/HASBonline/2017/3)",
+                        "text": "http://dx.doi.org/10.22013/HASBonline/2017/3)"
+                    }
+                ]
+            }
+        ],
+        "shortcode": "0852",
+        "spatialCoverage": [
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/6255148/europe.html",
+                "text": "Europe"
+            }
+        ],
+        "startDate": "2014-01-01",
+        "temporalCoverage": [
+            {
+                "__type": "URL",
+                "type": "Chronontology",
+                "url": "http://chronontology.dainst.org/period/RX2Rv7kcbLGk",
+                "text": "Roman Republic"
+            },
+            {
+                "__type": "URL",
+                "type": "Chronontology",
+                "url": "http://chronontology.dainst.org/period/ojQ3ix9NCdGq",
+                "text": "Roman imperial period"
+            },
+            {
+                "__type": "URL",
+                "type": "Chronontology",
+                "url": "http://chronontology.dainst.org/period/9ni4lSCi1app",
+                "text": "Late Antiquity II"
+            }
+        ],
+        "status": "Finished"
+    },
+    "datasets": [
+        {
+            "__id": "http://ns.dasch.swiss/repository#dsp-0852-dataset-000",
+            "__type": "Dataset",
+            "abstracts": [
+                {
+                    "en": "Catalogue of graves, grave monuments and grave inscriptions as testimonies of female literacy in the Roman Empire.\r -Collection of graves with writing utensils from the Latin West of the Roman Empire. It includes female, male and sexually indeterminate graves. \r -Funerary monuments remind us of the deceased long after the burial and create a certain image with which they are remembered. In this catalogue, grave monuments of women with writing implements as attributes from the Latin West are compiled. Due to the large number of sarcophagus reliefs, these are only included as a representative sample. Representations of Muses are excluded unless there is an identification with the deceased person.\r -Compilation of funerary inscriptions of women with a connection to literacy. The inscriptions mention activities that presuppose or make probable a certain mastery of reading and writing. This ranges from working as a reader or scribe to attending school and being praised by ones high education."
```

Comment:
Not sure if carriage return `\r` will be displayed as intended. Maybe a new line `\n` works better?